jinja2/utils.py

```diff
@@ -506,12 +506,14 @@ def current(self):
         """Returns the current item."""
         return self.items[self.pos]
 
-    def __next__(self):
+    def next(self):
```

Comment:
In Python 3, `implements_iterator` does nothing.  But in Python 2, it renames `__next__` to `next` (and deletes `__next__`).  <https://github.com/mitsuhiko/jinja2/blob/master/jinja2/_compat.py#L70>

If only a `__next__` method is explicitly defined, the Python 3 tests fail with:

```
    def test_cycler_nextmethod(self, env):
        items = 1, 2, 3
        c = Cycler(*items)
        for item in items + items:
            assert c.current == item
>           assert c.next() == item
E           AttributeError: 'Cycler' object has no attribute 'next'
```

I think the test's assertion is correct because the [docs](http://jinja.pocoo.org/docs/dev/templates/#cycler.next) imply that a `Cycler` has a method named `next()`.